Context.sublime-menu

```diff
@@ -5,35 +5,35 @@
         [
             {
                 "command": "lsp_symbol_references",
-                "caption": "Find Symbol References"
+                "caption": "Goto Reference…"
```

Comment:
Instead of `Goto Reference`, 
I would suggest `Find References`

Reply:
oh, now I see why you choose `Goto reference`, and the `...`.
But I would still consider `Find References` more suitable.
 
![image](https://user-images.githubusercontent.com/22029477/97912104-a05b7380-1d4c-11eb-8120-1c66e075165e.png)